error in configure step if BUILD_MPI=false

There is an error in configure step when not building MPI : 

```
CMake Error at src/CMakeLists.txt:11 (file):
  Error evaluating generator expression:

    $<TARGET_FILE_NAME:bendersmpi>

  No target "bendersmpi"
```

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -4,6 +4,12 @@ if(BUILD_TESTING)
 	add_subdirectory(tests)
 endif()
 
+if(USE_MPI)
+    set(BENDERSMPI_APP $<TARGET_FILE_NAME:bendersmpi>)    
+else()    
+    set(BENDERSMPI_APP "mpi_not_available")    
+endif()
+
 #configure file to define antares-solver executable
 configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/src_python/config.yaml.in  ${CMAKE_CURRENT_SOURCE_DIR}/src_python/config.yaml)
 

--- a/src/src_python/config.yaml.in
+++ b/src/src_python/config.yaml.in
@@ -1,5 +1,5 @@
 ANTARES             : @ANTARES_SOLVER_APP@
 MERGE_MPS           : $<TARGET_FILE_NAME:merge_mps>
-BENDERS_MPI         : $<TARGET_FILE_NAME:bendersmpi>
+BENDERS_MPI         : @BENDERSMPI_APP@
 BENDERS_SEQUENTIAL  : $<TARGET_FILE_NAME:benderssequential>
 LP_NAMER            : $<TARGET_FILE_NAME:lp_namer>

